hf-llm-inference/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ghcr.io/coreweave/ml-containers/torch:afecfe9-base-cuda12.0.1-torch2.0.0-vision0.15.1
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt upgrade -y && \
+    apt update && apt install -y python3 python3-pip git curl && \
+    apt clean
+
+RUN mkdir /app
+WORKDIR /app
+
+ARG COMMIT=master
```

Comment:
Try defaulting to [the commit `4ff78daa65b6a4a944c5f14f102e5f28770bcfcc` from your PR](https://github.com/coreweave/kubernetes-cloud/pull/303/commits/4ff78daa65b6a4a944c5f14f102e5f28770bcfcc) so it can build. Using a branch name as the default leads to weird things because of Dockerfile build layer caching; it will repeatedly build from the same commit as whatever was the `HEAD` on its first run (unless the cache is invalidated or cleared).